coldfront/core/project/models.py

```diff
@@ -80,9 +80,12 @@ def save(self, *args, **kwargs):
                     # begins after the project becomes active.
                     project_users = self.projectuser_set.filter(
                         status=pending_status)
+
                     for project_user in project_users:
+                        reason = project_user.projectuserjoinrequest_set.latest('created').reason
```

Comment:
In the unexpected case when a pending `ProjectUser` has no join request (e.g., `latest` evaluates to `None`), this will raise an exception. In that case, let's just use the default reason.